### PR TITLE
fix(CI): RHCLOUD-26746 Replace build-deploy-common library 

### PR DESCRIPTION
**Pull request by Victoremepunto**
https://github.com/RedHatInsights/cicd-tools/tree/aeb9fe8b13b4a5524014ada4b0d324020dd04289

## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [x] General Coding Practices

### CODE REVIEW

File: scripts/unit_test.sh [hunk 1]
```diff
@@ -24,28 +28,31 @@ DATABASE_USER="compliance"
...
-  podman pod rm -f "$COMPLIANCE_POD_ID" || true
+  cicd::container::cmd pod rm -f "$COMPLIANCE_POD_ID" || echo "couldn't delete pod ID: $COMPLIANCE_POD_ID"
 }
 
-trap "teardown_podman" EXIT SIGINT SIGTERM
+trap "teardown" EXIT SIGINT SIGTERM
 
-if ! COMPLIANCE_POD_ID=$(podman pod create --name "$POD_NAME"); then
+if ! COMPLIANCE_POD_ID=$(cicd::container::cmd pod create --name "$POD_NAME"); then
     exit 1
```
Comment (romanblanco) [inline scripts/unit_test.sh:43]: @Victoremepunto, there is no information provided for this `exit 1`, maybe some hint could be added:

```suggestion
    echo "Failed creating pod image"
    exit 1
```

Comment (skateman) [reply scripts/unit_test.sh:43]: That's useful 😁

Comment (skateman) [reply scripts/unit_test.sh:43]: It wasn't a joke

**Review by romanblanco — APPROVED** [review]
LGTM :+1:

**Pull request merged**